src/structs.md

```diff
@@ -17,3 +17,14 @@ fn main() {
     println!("{} is {} years old", peter.name, peter.age);
 }
 ```
+
+<details>
+Key Points: 
+
+* Structs work similar as they do in other languages, but note there is no `typedef` keyword or inheritance between structs. 
```

Comment:
WDYT about splitting this in multiple lines, e.g.:
* Structs work like in C or C++.
  * Like in C++, and unlike in C, no `typedef` is needed to define a type.
  * Unlike in C++, there is no inheritance between structs.

Reply:
I like it.